tools/build.py

```diff
@@ -128,7 +130,7 @@ def configure_build(arguments):
     return subprocess.call(cmake_cmd)
 
 def build_jerry(arguments):
-    return subprocess.call(['make', '--no-print-directory','-j', '-C', BUILD_DIR])
+    return subprocess.call(['make', '--no-print-directory','-j', str(arguments.parallel_build), '-C', BUILD_DIR])
```

Comment:
It will prints the 'make: the `-j' option requires a positive integral argument' message, and the build will fail.